Session 3/task1.js

```diff
@@ -0,0 +1,17 @@
+/*
+http://www.codewars.com/kata/prefill-an-array
+*/
+
+function prefill(n, v) {
+  
+  if(n != parseInt(n, 10) || n<0 ){
```

Comment:
user isNaN function

Reply:
Here in this problem, when n is "0" or when n is "1" or something like that it should be converted to integer to pass the test cases. So we have to parse it to integer to make it work.